modules/transfer-frontend/security.tf

```diff
@@ -7,14 +7,14 @@ resource "aws_security_group" "lb" {
     protocol    = "tcp"
     from_port   = 443
     to_port     = 443
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = local.environment == "intg" ? var.ip_whitelist : ["0.0.0.0/0"]
```

Comment:
Minor thing, but is the `local.environment == "intg"` necessary? Could we make the `0.0.0.0/0` the variable default?

I think it would make it a bit easier to follow what's going on. For example if we were to pass in an `ip_whitelist` variable in staging, this check means it wouldn't work. We'd have to track down this conditional to find the problem.

Reply:
updated as suggested, with conditional in root